subsys/bluetooth/audio/ascs.c

```diff
@@ -338,11 +337,24 @@ static void ascs_iso_disconnected(struct bt_iso_chan *chan, uint8_t reason)
 		BT_WARN("No callback for stopped set");
 	}
 
-	ascs_ep_set_state(ep, BT_AUDIO_EP_STATE_QOS_CONFIGURED);
+	if (ep->status.state == BT_AUDIO_EP_STATE_RELEASING) {
+		ascs_ep_set_state(ep, BT_AUDIO_EP_STATE_CODEC_CONFIGURED);
+	} else { /* Streaming or Enabling */
```

Comment:
Should we have proper state checks, instead of a comment (`/* Streaming or Enabling */`)?

Reply:
> Should we have proper state checks, instead of a comment (`/* Streaming or Enabling */`)?

I will remove the comment as the CIS can exist in disabling and qos configured state as well